### PR TITLE
BREAKING CHANGE: show notification message

### PR DESCRIPTION
**Pull request by GusBaamonde**
- Adapt the user-menu container to be able to show it by class and not by ID, reusing the style of the container

panel-notificaciones

### CODE REVIEW

File: assets/js/main.js [hunk 1]
```diff
@@ -11,8 +11,8 @@ $("body").on({
   }
 }, ".menu-main .submenu-item");
 
-$("body").on('click', '#user-menu--open', function() {
-  $('.user-menu').toggleClass('open');
+$("body").on('click', '.user-menu--open', function() {
```
Comment (andresmoschini) [inline assets/js/main.js]: It is not a breaking change?

Comment (GusBaamonde) [reply assets/js/main.js]: you're right

**Comment by andresmoschini** [comment]
Relacionado con: https://github.com/FromDoppler/doppler-webapp/pull/148

**Pull request merged**